lib/http_fuzzer/engine.py

```diff
@@ -338,7 +350,9 @@ def sample_event_key_evaluator(response, payload, value):
     """
     try:
         if value != '':
-            exec ("value = " + value)
+            # value1 = ''
```

Comment:
same error in flake8 and pylint - crash with:

ib/http_fuzzer/engine.py:355:21: F405 'value1' may be undefined

Reply:
i defined global value1